src/js/hooks/inboundV2/addItems/useInboundAddItemsValidation.js

```diff
@@ -0,0 +1,62 @@
+import { z } from 'zod';
+
+import useTranslate from 'hooks/useTranslate';
+
+const useInboundV2Validation = () => {
+  const translate = useTranslate();
+
+  const lineItemSchema = z.object({
+    palletName: z.string().optional(),
+    boxName: z.string().optional(),
+    product: z.object({
+      id: z.string(),
+      value: z.string(),
+      label: z.string(),
+    }).optional().nullable(),
+    lotNumber: z.string().optional(),
+    expirationDate: z.string().optional().nullable(),
+    quantityRequested: z.number()
+      .min(0, translate('react.stockMovement.error.enterQuantity.label', 'Enter proper quantity'))
+      .refine((value) => value >= 0, {
+        message: translate('react.stockMovement.error.enterQuantity.label', 'Enter proper quantity'),
+      })
+      .optional(),
+    recipient: z.object({
+      id: z.string(),
+      value: z.string(),
+      label: z.string(),
+    }).optional().nullable(),
+  })
+    .refine((data) => !(data.boxName && !data.palletName), {
+      message: translate('react.stockMovement.error.boxWithoutPallet.label', 'Please enter Pack level 1 before Pack level 2'),
+      path: ['boxName'],
+    })
+    .refine((data) => {
+      const isValid = !(data.expirationDate && (!data.lotNumber || data.lotNumber === ''));
+      return isValid;
```

Comment:
```suggestion
      return !(data.expirationDate && (!data.lotNumber || data.lotNumber === ''));

```

\+ `!data.lotNumber || data.lotNumber === ''` <- isn't the `data.lotNumber === ""` included in `!data.lotNumber`?